tests/spec/core/examples-spec.js

```diff
@@ -51,7 +51,7 @@ describe("Core — Examples", () => {
     expect(marker.querySelector(".example-title").textContent).toBe(": EX");
     expect(example.getAttribute("title")).toBeNull();
     expect(example.textContent.trim()).toBe(
-      "Example 1: EX\n  \n{\n  CONTENT\n}"
+      "Example 1:  EX\n  \n{\n  CONTENT\n}"
```

Comment:
Why does Chrome report twice? It doesn't seem to be a bug in this script.

Reply:
Yeah. I am confused too. Probably something temporary. The one below is reported correctly.
We can fix later if we see it in real life again